Sources/Symbolicator/SymbolicatorApp.swift

```diff
@@ -3,7 +3,7 @@ import ArgumentParser
 import GenericJSON
 
 @main
-struct SymbolicatorApp: ParsableCommand {
+struct Symbolicator: ParsableCommand {
```

Comment:
I'd prefer the `@main` class to be called *App.

Reply:
the reason I renamed was that help documentation was autogenerated like that:

`USAGE: symbolicator-app <input-file-argument> [--dsym-file <dsym-file>] [--app-name <app-name>] [--arch <arch>] [--json]`